apps/src/code-studio/progress.js

```diff
@@ -78,6 +78,8 @@ progress.showDisabledBubblesAlert = function() {
  *   user, null otherwise
  * @param {boolean} stageExtrasEnabled Whether this user is in a section with
  *   stageExtras enabled for this script
+ * @param {boolean} isLessonExtras Boolean indicating we are not on a script
```

Comment:
super minor NIT, but would be good to include your `currentPageNumber` annotation from above here and at `initializeStoreWithProgress`

Reply:
👍